Serverless troubleshooting guide (#13163)
DataDog/documentation: content/en/serverless/installation/dotnet.md

- Data stores: DynamoDB, S3, RDS, etc.

To collect logs from non-Lambda AWS resources, install and configure the [Datadog Forwarder][2] to subscribe to each of your managed resource CloudWatch log groups.

## Troubleshooting

> If you have trouble collecting the monitoring data after following the instructions above, refer to the [serverless monitoring troubleshooting guide][8] for help.

Reviewer: ```suggestion
If you have trouble collecting monitoring data after following the instructions above, see the [serverless monitoring troubleshooting guide][8].

```
